### PR TITLE
Fix eslint ignoring the .erb directory by default

### LINKED ISSUE #3140: Files in the `.erb` directory don't seem to be linted
## Prerequisites

- [X] Using **yarn**
- [X] Using an up-to-date [`main` branch](https://github.com/electron-react-boilerplate/electron-react-boilerplate/tree/main)
- [ ] Using latest version of devtools. [Check the docs for how to update](https://electron-react-boilerplate.js.org/docs/dev-tools/)
- [ ] Tried solutions mentioned in [#400](https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues/400)
- [ ] For issue in production release, add devtools output of `DEBUG_PROD=true npm run build && npm start`

## Expected Behavior
All files in the project, including the scripts and configs in the `.erb` directory are linted.

## Current Behavior
The files in the `.erb` directory don't seem to be linted. If I add `!.erb` to `.eslintignore`, I get linting errors:

```
/home/kgy/src/electron-react-boilerplate/.erb/configs/webpack.config.renderer.dev.ts
   11:1   error    `@pmmmwh/react-refresh-webpack-plugin` import should occur before import of `./webpack.config.base`  import/order
   21:29  warning  Forbidden non-null assertion                                                                         @typescript-eslint/no-non-null-assertion
  152:3   error    Do not use "@ts-ignore" because it alters compilation errors                                         @typescript-eslint/ban-ts-comment
  171:53  warning  Forbidden non-null assertion                                                                         @typescript-eslint/no-non-null-assertion

/home/kgy/src/electron-react-boilerplate/.erb/scripts/check-native-dep.js
  41:34  error  Replace `'cd·./release/app·&&·npm·install·your-package'` with `⏎··'cd·./release/app·&&·npm·install·your-package'⏎`  prettier/prettier

/home/kgy/src/electron-react-boilerplate/.erb/scripts/clean.js
  2:26  error  Unexpected use of file extension "ts" for "../configs/webpack.paths.ts"       import/extensions
  3:1   error  `process` import should occur before import of `../configs/webpack.paths.ts`  import/order

/home/kgy/src/electron-react-boilerplate/.erb/scripts/electron-rebuild.js
  1:8  warning  'path' is defined but never used  @typescript-eslint/no-unused-vars

/home/kgy/src/electron-react-boilerplate/.erb/scripts/link-modules.ts
  4:7   error  Use object destructuring  prefer-destructuring
  5:7   error  Use object destructuring  prefer-destructuring
  5:59  error  Insert `;`                prettier/prettier

/home/kgy/src/electron-react-boilerplate/.erb/scripts/notarize.js
  10:26  error  Replace `"true"` with `'true'`                                                                                                                                                                  prettier/prettier
  16:18  error  Replace `'Skipping·notarizing·step.·APPLE_ID·and·APPLE_ID_PASS·env·variables·must·be·set'` with `⏎······'Skipping·notarizing·step.·APPLE_ID·and·APPLE_ID_PASS·env·variables·must·be·set'⏎····`  prettier/prettier
```

## Steps to Reproduce
1. `yarn install`
2. Add `!.erb` to `.eslintignore`
3. `yarn lint`

## Context
I am adding some configs in the `.erb` directory, and would like them to be linted.

## Your Environment
- Node version : `v16.13.2`
- electron-react-boilerplate version or branch : 9ef120923cdfa85043aed2f7785c4ce9dac57f62

### PR DESCRIPTION
**Pull request by gyorgy-kurucz-vacuum**
Fixes #3140

### CODE REVIEW

File: .erb/configs/.eslintrc [hunk 1]
```diff
@@ -2,6 +2,8 @@
   "rules": {
     "no-console": "off",
     "global-require": "off",
-    "import/no-dynamic-require": "off"
+    "import/no-dynamic-require": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
+    "@typescript-eslint/ban-ts-comment": "off"
```
Comment (amilajack) [inline .erb/configs/.eslintrc:7]: Why this rule? Seems like there aren't any ts bans in this PR

File: .erb/configs/.eslintrc [hunk 2]
```diff
@@ -2,6 +2,8 @@
   "rules": {
     "no-console": "off",
     "global-require": "off",
-    "import/no-dynamic-require": "off"
+    "import/no-dynamic-require": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
```
Comment (amilajack) [inline .erb/configs/.eslintrc:6]: Its ignore on a per-line level (ex. `// eslint-disable-next-line`)

**Review by amilajack — CHANGES_REQUESTED** [review]
Pretty much good to go, just a few minor comments!

File: .erb/configs/webpack.config.renderer.dev.ts [hunk 3]
```diff
@@ -149,6 +150,7 @@ const configuration: webpack.Configuration = {
     __filename: false,
   },
 
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-ignore
```
Comment (gyorgy-kurucz-vacuum) [inline .erb/configs/webpack.config.renderer.dev.ts:154]: This is quite ugly, but it's not obvious to me how to get rid of this `@ts-ignore`.

File: .erb/configs/.eslintrc (hunk 2 above)
Comment (gyorgy-kurucz-vacuum) [reply .erb/configs/.eslintrc]: Fixed :+1: I removed these and added `eslint-disable-next-line`s instead.

**Comment by gyorgy-kurucz-vacuum** [comment]
I added a separate fixup commit so the changes are easily visible, it will need to be squashed before merging.

File: .erb/configs/.eslintrc (hunk 1 above)
Comment (gyorgy-kurucz-vacuum) [reply .erb/configs/.eslintrc]: There is a `@ts-ignore` in [.erb/configs/webpack.config.renderer.dev.ts](https://github.com/electron-react-boilerplate/electron-react-boilerplate/pull/3165/files/8c3be67f48677c37232fc8aa86c00668ac132905#diff-98d366b10a60c9624cca34f2d0e8bc99b2e130d73fa9aadca5bf00631733fe73)

**Review by amilajack — APPROVED** [review]

**Comment by amilajack** [comment]
Great stuff @gyorgy-kurucz-vacuum! 🥳

**Pull request merged**